components/FinalVideoEditor.tsx

```
@@ -2,18 +2,10 @@
-  Dialog,
-  DialogContent,
-  DialogDescription,
-  DialogFooter,
-  DialogHeader,
-  DialogTitle,
-} from '@/components/ui/dialog';
-import { Loader2, Play } from 'lucide-react';
+import { Play } from 'lucide-react';
+import { getPresetBezier } from '@/lib/easing-presets';
+import { useVideoPlayback } from '@/hooks/useVideoPlayback';
 import { getPresetBezier } from '@/lib/easing-presets';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'getPresetBezier'. Consider to delete it or rename it.

'getPresetBezier' is defined here:


(lint/suspicious/noRedeclare)